python/README.md

```diff
@@ -1,171 +1,146 @@
-# Coding Style
+# Python Coding Style 🐍
 
-Our Python coding style is [PEP8](http://www.python.org/dev/peps/pep-0008/).
+**TLDR**: Configure your text editor to use [`flake8`](https://flake8.readthedocs.org/en/latest/):
 
-We have an additional standard for import order:
+**Sublime Text**
 
-    # Standard libs first in alpha order.
-    import abc
-    import urlparse
+Install [`flake8`](https://flake8.readthedocs.org/en/latest/#quickstart) then
+using [Package Control](https://packagecontrol.io/installation),
+install [SublimeLinter](http://www.sublimelinter.com/en/latest/installation.html)
+and [SublimeLinter-flake8](https://github.com/SublimeLinter/SublimeLinter-flake8).
 
-    # Third party modules.
-    import requests
+## [PEP8](https://www.python.org/dev/peps/pep-0008/)
 
-    # Project modules (using relative paths where necessary)
-    from . import mymodule
+We follow [PEP8](http://www.python.org/dev/peps/pep-0008/) with an additional
+standard for ordering imports:
 
-It's easy to check your code for PEP8 compliance:
+```python
+# First, standard libraries ordered alphabetically.
+import abc
+import urlparse
 
-**Checking from the command line**
+# Second, third-party libraries.
+import requests
 
-Install the `pep8` package and run it on one or more directories or files:
+# Third, project libraries, using using relative paths where applicable.
+from . import mymodule
+```
 
-    $ pip install pep8
-    $ # check an individual file
-    $ pep8 portal/core/admin.py 
-    $ # check a whole directory
-    $ pep8 portal/mailers
+### PEP8 on the Command Line
 
-**Checking in IDEs/editors**
+Install the [`pep8`](https://pypi.python.org/pypi/pep8) package and run it on
```

Comment:
As one of the recent developments, the Python community established the PyCQA - the Python Code Quality Authority - which is now managing a set of linting tools. As part of their work they renamed the tool `pep8` to `pycodestyle` ([discussion kicked off by Guido](https://github.com/PyCQA/pycodestyle/issues/466)). So this should be `pycodestyle` instead of `pep8` now.